uint16 front_right
uint16 front_left
uint16 rear_right
uint16 rear_left
